user-service/src/unit-tests/java/uk/gov/gchq/palisade/service/user/service/NullUserServiceTest.java

```diff
@@ -37,6 +36,6 @@ public void getUser() {
     public void addUser() {
         User user = new User().userId("testUser");
         User actual = nullUserService.addUser(user);
-        assertThat(user, is(actual));
+        assertThat(actual.getUserId().getId()).isEqualTo(user.getUserId().getId());
```

Comment:
worth testing the whole user, saves us in the rare case that we set some vars accidentally when caching the user
```suggestion
        assertThat(actual).isEqualTo(user);
```